kafka-init/pom.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>strimzi</artifactId>
+    <groupId>io.strimzi</groupId>
+    <version>0.5.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>kafka-init</artifactId>
```

Comment:
The package is init/kafka but the artifact id is the other way around. Maybe they should be consistent.

Reply:
Because I was thinking about having a package for other possible init applications (for ZK and Connect) so something like `io.strimzi.init.kafka`, `io.strimzi.init.zookeeper` and so on. Do you agree if I'll change the artifact-id, so the subproject name and so on using `init-kafka` ? What do you think ?

Reply:
That seems fine to me.